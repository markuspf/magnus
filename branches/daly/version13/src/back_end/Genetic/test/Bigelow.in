<g1,g2,g3,g4,g5>
[g3^( g4^-1 g5^2 g2 g1^-2 ), g3^( g4 g5^-1 g2^-1 g1 )],
