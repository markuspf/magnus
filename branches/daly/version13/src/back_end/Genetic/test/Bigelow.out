Lengths: 

y1 :  7249
y2 :  4273
y3 :  6817
y4 :  6817
y5 :  2543
y6 :  6385

y1 * y2 :  2976
y2 * y3 :  11090
y3 * y4 :  6818
y4 * y5 :  9360
y5 * y6 :  3842

y1 * y2 * y3 :  3843
y2 * y3 * y4 :  11091
y3 * y4 * y5 :  9361
y4 * y5 * y6 :  3841

y1 * y2 * y3 * y4 :  3844
y2 * y3 * y4 * y5 :  13634
y3 * y4 * y5 * y6 :  2980

y1 * y2 * y3 * y4 * y5 :  6387
y2 * y3 * y4 * y5 * y6 :  7253

y1 * y2 * y3 * y4 * y5 * y6 :  6